src/sklearn_evaluation/plot/classification.py

```diff
@@ -317,24 +321,28 @@ def feature_importances(
     .. plot:: ../examples/feature_importances.py
 
     """
-    if data is None:
-        raise ValueError(
-            "data is needed to plot feature importances. "
-            "When plotting using the evaluator you need to pass "
-            "an estimator "
-        )
+    try:
+        if data is None:
+            raise ValueError(
```

Comment:
I think we missed this one

Reply:
Since `compute.feature_importances(data, top_n, feature_names)` throws a ValueError from a 3rd party lib, I wrapped everything with try/catch (didn't want to raise PloomberValueError twice). I can change it to something like this

```python
    if data is None:
        raise PloomberValueError(
                "data is needed to plot feature importances. "
                "When plotting using the evaluator you need to pass "
                "an estimator "
        )
    try:
        # If no feature_names is provided, assign numbers
        res = compute.feature_importances(data, top_n, feature_names)

        ax = bar.plot(
            res.importance,
            orientation,
            res.feature_name,
            error=None if not hasattr(res, "std_") else res.std_,
        )
        ax.set_title("Feature importances")
        return ax

    except ValueError as e:
        raise PloomberValueError(e)

```
what do you think?